lib/analysis.js

```diff
@@ -61,13 +68,23 @@ AnalysisFactory.prototype.create = function(configuration, definition, callback)
                 factory.create(JSON.parse(JSON.stringify(definition)), done);
             },
             function analysis$validate(rootNode, done) {
+                logger = logger.child({ analysis: rootNode.id() }, true);
+
+                databaseService.setLogger(logger);
+
+                logger.info({ username: configuration.user });
+
                 if (!validator.isValid(rootNode)) {
                     return done(new Error('Invalid analysis, cycle found'));
                 }
 
                 return done(null, new Analysis(rootNode));
             },
             function analysis$register(analysis, done) {
+                analysis.getNodes().map(function (node) {
+                    logger.info({ node: node.id(), type: node.getType() });
```

Comment:
What about adding here the username and removing the other entry, the one that only has the username?